tools/localizations/cn.csv

```diff
@@ -56,4 +56,8 @@ Aqara Smart Smoke Detector,Aqara烟雾报警器
 Aqara Smart Plug T1,Aqara 智能插座 T1 (国标)
 Aqara Wireless Mini Switch T1,Aqara 无线开关 T1
 "Aqara Wireless Remote Switch E1 (Single Rocker)",Aqara 无线开关 E1 (贴墙式单键版)
-"Aqara LED Light Bulb T1 (Tunable White)",Aqara LED灯泡 T1 (可调色温)
\ No newline at end of file
+"Aqara LED Light Bulb T1 (Tunable White)",Aqara LED灯泡 T1 (可调色温)
+Aqara Smart Wall Switch H1 EU (With Neutral, Single Rocker),Aqara智能墙壁开关H1 EU (零火线单键版)
+Aqara Smart Wall Switch H1 EU (With Neutral, Double Rocker),Aqara智能墙壁开关H1 EU (零火线双键版)
+Aqara Smart Wall Switch H1 EU (No Neutral, Single Rocker),Aqara智能墙壁开关H1 EU (单火线单键版)
+Aqara Smart Wall Switch H1 EU (No Neutral, Double Rocker),Aqara智能墙壁开关H1 EU (单火线双键版)
```

Comment:
because of the parens in the name, quotes are required

Reply:
ok